src/MessageFormatter.php

```diff
@@ -70,7 +70,8 @@ public function format(
     ): string {
         $cache = [];
 
-        return \preg_replace_callback(
+        /** @var string $message */
+        $message = \preg_replace_callback(
```

Comment:
Can we use the suggestion of @GrahamCampbell with quick return annotation here?

Reply:
Thanks, missed this one! Fixed in 44148c15357f5601487877f3d8bf8a97a8c4b1af